.travis.yml

```diff
@@ -43,12 +43,14 @@ script:
     fi
   - |
     if [ -v PRODUCT_TESTS ]; then
-      presto-product-tests/bin/run_on_docker.sh singlenode -x quarantine,big_query,storage_formats,mysql_connector,postgresql_connector,profile_specific_tests
-      presto-product-tests/bin/run_on_docker.sh singlenode-kerberos-hdfs-impersonation -g storage_formats,cli,hdfs_impersonation,authorization
+      presto-product-tests/bin/run_on_docker.sh \
```

Comment:
I would rather let run them both. And than return `1` status if either of them failed.